crate/src/lib.rs

```diff
@@ -216,14 +232,17 @@ impl Universe {
     }
 
     pub fn new(width: i32, height: i32) -> Universe {
+        // seed from random.org
+        let mut rng: Isaac64Rng = SeedableRng::seed_from_u64(0x734f6b89de5f83cc);
```

Comment:
the game is now deterministic - not sure if I have an opinion on that, but interesting

Reply:
That could be a good thing if you want to implement snapshots like suggested here: https://www.reddit.com/r/rust/comments/a6v0p1/sandspiel_a_falling_sand_game_built_in_rustwebgl/ebyyt00/

Reply:
I do want that, main concern is squeezing the extra buttons into the layout
and having them be self explanatory .